### PR TITLE
Fix docstrings of regex template filters

### PR DESCRIPTION
**Pull request by happz**
They we a bit messy, especially examples.

Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

File: tests/integration/__init__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from requre.helpers.requests_response import RequestResponseHandling
 
 import tmt.utils
+import tmt.utils.git
```
Comment (martinhoyer) [inline tests/integration/__init__.py:10]: hmm, isn't this redundant?

**Review by martinhoyer — APPROVED** [review]

**Review by KwisatzHaderach — APPROVED** [review]

**Comment by happz** [comment]
Added `blocked` label: depends on a branch that's been pushed out to 1.36, the rebase would be a mess, and it's not critical.

File: tests/integration/__init__.py (hunk 1 above)
Comment (happz) [reply tests/integration/__init__.py:9]: Addressed in 23590472

**Review by martinhoyer — APPROVED** [review]

**Pull request merged**